userscript-styles/WKED-self-study.css

```
@@ -12,72 +12,72 @@
 /* See https://community.wanikani.com/t/userscript-self-study-quiz/13191 */
 
 @-moz-document domain("www.wanikani.com") {
-#wkof_ds #ss_quiz {
-  border-color: var(--ED-surface-2, #282828);
-  background-color: var(--ED-surface-2, #282828);
-}
+  #wkof_ds #ss_quiz {
+    border-color: var(--ED-surface-2, #282828);
+    background-color: var(--ED-surface-2, #282828);
+  }
 
-#wkof_ds #ss_quiz .titlebar {
-  background-color: var(--ED-surface-2, #282828);
-  color: var(--ED-text-color, #eeeeee);
-}
+  #wkof_ds #ss_quiz .titlebar {
+    background-color: var(--ED-surface-2, #282828);
+    color: var(--ED-text-color, #eeeeee);
+  }
 
-#wkof_ds #ss_quiz .statusbar {
-  background-color: var(--ED-surface-2, #282828);
-  color: var(--ED-text-color, #eeeeee);
-}
+  #wkof_ds #ss_quiz .statusbar {
+    background-color: var(--ED-surface-2, #282828);
+    color: var(--ED-text-color, #eeeeee);
+  }
 
-#wkof_ds #ss_quiz .answer {
-  background-color: var(--ED-surface-3, #303030);
-}
+  #wkof_ds #ss_quiz .answer {
+    background-color: var(--ED-surface-3, #303030);
+  }
 
-#wkof_ds #ss_quiz .answer input {
-  background-color: var(--ED-surface-5, #bababa);
-  border-color: var(--ED-surface-4, #535353);
-  --ED-text-color: var(--ED-inverted-text, #151515);
-}
+  #wkof_ds #ss_quiz .answer input {
+    background-color: var(--ED-surface-5, #bababa);
+    border-color: var(--ED-surface-4, #535353);
+    --ED-text-color: var(--ED-text-inv, #151515);
+  }
 
-#ss_quiz[data-itype="radical"] .qwrap,
-#ss_quiz .summary .que[title~="Radical"] {
-  background-color: var(--ED-radical-clr, hsl(1, 39%, 44%));
-}
+  #ss_quiz[data-itype="radical"] .qwrap,
+  #ss_quiz .summary .que[title~="Radical"] {
+    background-color: var(--ED-radical, hsl(1, 39%, 44%));
+  }
 
-#ss_quiz[data-itype="kanji"] .qwrap,
-#ss_quiz .summary .que[title~="Kanji"] {
-  background-color: var(--ED-kanji-clr, hsl(225, 23%, 44%));
-}
+  #ss_quiz[data-itype="kanji"] .qwrap,
+  #ss_quiz .summary .que[title~="Kanji"] {
+    background-color: var(--ED-kanji, hsl(225, 23%, 44%));
+  }
 
-#ss_quiz[data-itype="vocabulary"] .qwrap,
-#ss_quiz .summary .que[title~="Vocabulary"] {
-  background-color: var(--ED-vocab-clr, hsl(166, 36%, 32%));
-}
+  #ss_quiz[data-itype="vocabulary"] .qwrap,
+  #ss_quiz .summary .que[title~="Vocabulary"] {
+    background-color: var(--ED-vocab, hsl(166, 36%, 32%));
+  }
 
-#ss_quiz[data-atype="reading"] .atype {
-  color: var(--ED-text-color, #eeeeee);
-  text-shadow: none;
-  border: none;
-  background-color: var(--ED-reading-clr, #282828);
-}
+  #ss_quiz[data-atype="reading"] .atype {
+    color: var(--ED-text-color, #eeeeee);
```

`--ED-text-color` no longer exists
```suggestion
    color: var(--ED-text, #eeeeee);
```